Assignment1/src/Main.java

```diff
@@ -0,0 +1,57 @@
+import java.util.*;
+
+public class Main {
```

Comment:
Formatting of the code is as important as the code itself. I see no formatting done on the code.